src/lib/registration/ui/addon_selection_base_dialog.rb

```diff
@@ -76,12 +86,12 @@ def addon_selected?(_addon)
       # @return [Yast::Term] the main UI dialog term
       def content
         VBox(
-          VStretch(),
           Left(Heading(heading)),
+          Left(CheckBox(Id(:filter_beta), Opt(:notify),
```

Comment:
For the alpha / pre-alpha, I don't think that we need this for SP2; hiding the beta check-box if there are none available is something that we should IMO handle before GM (e.g. next sprint), the way should be, however, discussed with UX experts (customers could ask "where is the check box?")